front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -604,19 +547,15 @@ const AdminProjectsProjectGeneral = ({
         />
 
         <GeographicAreaInputs
-          areaType={areaType}
-          areasOptions={areasOptions}
-          areasValues={areasValues}
-          handleAreaTypeChange={handleAreaTypeChange}
-          handleAreaSelectionChange={handleAreaSelectionChange}
+          areaIds={projectAttrs.area_ids}
+          onProjectAttributesDiffChange={handleProjectAttributeDiffOnChange}
         />
 
-        {/* lint error, @brent please look into this */}
-        {/* <Outlet
+        <Outlet
           id="app.components.AdminPage.projects.form.additionalInputs.inputs"
           projectAttrs={projectAttrs}
           onProjectAttributesDiffChange={handleProjectAttributeDiffOnChange}
-        /> */}
```

Comment:
Uncommented and didn't have (linter) issues with this. Lmk if I missed something.